src/store/lots/lotsReducer.ts

```diff
@@ -9,21 +9,35 @@ const initialState: any = {
 const lotsReducer = (state = initialState, action: LotsActions): any => {
   switch (action.type) {
     case LotsActionsType.GET_LOTS:
-      return { ...state, lots: action.payload, isLoading: false, isError: false };
+      return { ...state, lots: action.payload, isLoading: false };
 
     case LotsActionsType.REMOVE_LOT:
       return {
         ...state,
         lots: state.lots.filter((lot: any) => lot.id !== action.payload),
         isLoading: false,
-        isError: false,
+      };
+
+    case LotsActionsType.CREATE_LOT:
+      return {
+        ...state,
+        lots: state.lots.push(action.payload),
+        isLoading: false,
+      };
+
+    case LotsActionsType.UPDATE_LOT:
+      return {
+        ...state,
+        // TODO write update action
+        lots: state.lots, // state.lots.find((lot: any) => lot.id === action.payload.id),
+        isLoading: false,
```

Comment:
it is better to create separate action/reducer for loading state, plus specify what is actually loading. e.g `isLostsLoading`